glusterlib/playbook_gen.py

```diff
@@ -136,19 +136,15 @@ def move_templates_to_playbooks(self):
         /tmp/playbooks
         '''
         # Is the templates present as a part of ansible installation?
-        templates_path_pkg = '/usr/share/ansible/ansible-glusterfs/templates'
-        # Or is it present in the source directory or installed via setuptools
-        templates_path_bk = self.get_file_dir_path(self.uppath(__file__, 2),
-                                                   'templates')
```

Comment:
On Sun, Aug 9, 2015 at 11:09 AM, Nandaja Varma <notifications@github.com>
wrote:

> In glusterlib/playbook_gen.py
> <https://github.com/nandajavarma/gdeploy/pull/12#discussion_r36584729>:
>
> > @@ -136,19 +136,15 @@ def move_templates_to_playbooks(self):
> >          /tmp/playbooks
> >          '''
> >          # Is the templates present as a part of ansible installation?
> > -        templates_path_pkg = '/usr/share/ansible/ansible-glusterfs/templates'
> > -        # Or is it present in the source directory or installed via setuptools
> > -        templates_path_bk = self.get_file_dir_path(self.uppath(__file__, 2),
> > -                                                   'templates')
>
> Also the templates has a bk directory keeping in mind that the templates
> can be stored in the current working directory and can be run. For someone
> who is developing or using this without installing the whole package, this
> is much easier a way. If I change a playbook, installing it every time to
> test seems to be a hard way.
>

I made a mistake here. I mistook templates to ansible modules. Don't submit
this patch yet.
Just change the path of templates directory in message and in the code.
Leave the _bk alone.

:-)

My bad.